### PR TITLE
[4.0] [a11y] add sr-only text to first column of table head

### PR DESCRIPTION
**Pull request by hans2103**
Pull Request for Issue #30214 .

### Summary of Changes

This PR has influence on the installation of Joomla.
Once you get on _ Install Additional Languages_  it will add a screen reader only text to the table head of the first column of the table shown.

### Testing Instructions

- Install Joomla 4
- Press _Install Additional Languages_
- Inspect element on the first column of the table.
- Apply the PR and inspect the element again.

Schermafdruk 2020-08-02 20 54 45

### Actual result BEFORE applying this Pull Request

Schermafdruk 2020-08-02 20 41 29

### Expected result AFTER applying this Pull Request

Schermafdruk 2020-08-02 20 56 01

### CODE REVIEW

**Comment by brianteeman** [comment]
While I obviously disagree with the need for this it is not correct markup.

If it is a table heading then it must be a th and not a td. You can also not have a scope on a td

**Comment by hans2103** [comment]
> While I obviously disagree with the need for this it is not correct markup.
> 
> If it is a table heading then it must be a th and not a td. You can also not have a scope on a td

thank you for your feedback. I will update my PR.

File: installation/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -110,6 +110,7 @@ INSTL_REMOVE_INST_FOLDER="Are you sure you want to delete? Confirming will perma
 
 ; Languages view
 INSTL_LANGUAGES="Install Additional Languages"
+INSTL_LANGUAGES_COLUMN_HEADER_LANGUAGE_SELECT="Select Language"
```
Comment (Quy) [inline installation/language/en-GB/joomla.ini]: Please alpha sort.

Comment (hans2103) [reply installation/language/en-GB/joomla.ini]: done

**Comment by brianteeman** [comment]
The code now does what it says it will do

**Comment by zwiastunsw** [comment]
I have tested this item :white_check_mark: successfully on 445984c999d4f82e0f65051e3f15c2d5b53cd870<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30262">issues.joomla.org/tracker/joomla-cms/30262</a>.</sub>

**Comment by armenos** [comment]
I have tested this item ✅ successfully

**Comment by armenos** [comment]
I have tested this item :white_check_mark: successfully on 445984c999d4f82e0f65051e3f15c2d5b53cd870<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30262">issues.joomla.org/tracker/joomla-cms/30262</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30262">issues.joomla.org/tracker/joomla-cms/30262</a>.</sub>

**Pull request merged**